it/uk/gov/hmrc/fileupload/ITApplicationComponents.scala

```diff
@@ -43,6 +43,6 @@ trait ITApplicationComponents extends OneServerPerSuite with MongoSpecSupport {
 
 }
 
-class TestApplicationModule(context: Context) extends ApplicationModule(context = context) {
+class ITestApplicationModule(context: Context) extends ApplicationModule(context = context) {
```

Comment:
Sorry this is quite poorly named @konradwudkowski. It is I for Integration because IntelliJ does not like that there is another TestApplicationModule in ApplicationComponents (for non integration tests). I will rename once I think of what to name it to :)